src/components/shared/CopyableAddressV2/CopyableAddressV2.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useCallback } from 'react';
+import clsx from 'clsx';
+import { splitAddress } from '~utils/strings';
+
+import { Address } from '~types';
+
+import MaskedAddress from '~shared/MaskedAddress';
+import useClipboardCopy from '~hooks/useClipboardCopy';
+import Icon from '~shared/Icon';
+// Note that we use CSS here because MaskedAddress css file will override font-normal by tailwind due to injection order
+import styles from './CopyableAddressV2.css';
+
+interface Props {
+  /** Address to display */
+  children: Address;
```

Comment:
I don't like this prop being called children yet not accepting `React.ReactNode` or similar

Reply:
I actually don't mind this, because it disabled you from passing in JSX components, it limits you only to string children, which isn't bad practice. Maybe it's not applicable to this component, I can just turn this into a prop instead